footbar e faq

lucassamagalhaes opened the pull request:
Gente, fiz o footbar com a data atual e criei uma rota para o faq ao clicar em Ajuda na navbar.

Commit:
footbar e faq

lucwars commented:
Eu tava fazendo o forte kkkk, mas sem problemas vlw cara

Obter o Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: Lucas Magalhães do Nascimento <notifications@github.com>
Enviado: Saturday, March 14, 2020 11:13:31 AM
Para: lucwars/Desenvolvimento-web2 <Desenvolvimento-web2@noreply.github.com>
Cc: LUCAS ALVES <lucwars@hotmail.com>; Review requested <review_requested@noreply.github.com>
Assunto: [lucwars/Desenvolvimento-web2] footbar e faq (#1)


Gente, fiz o footbar com a data atual e criei uma rota para o faq ao clicar em Ajuda na navbar.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/lucwars/Desenvolvimento-web2/pull/1

Commit Summary

  *   footbar e faq

File Changes

  *   M teste/Spotify/package-lock.json<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-d6cd016436097c4e49a4f2abf430edbb> (58)
  *   M teste/Spotify/package.json<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-433a21331cf187b2b77ede3cd82d2d04> (4)
  *   M teste/Spotify/src/app/app-routing.module.ts<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-5a38fb200a90a6ca8e82fff2b3597d89> (15)
  *   M teste/Spotify/src/app/app.component.html<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-cf07c06a346366939b7576a913d6eb19> (3)
  *   M teste/Spotify/src/app/app.module.ts<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-c598003e335a06d9fd56a4a3f1f19189> (6)
  *   A teste/Spotify/src/app/faq/faq.component.css<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-e4be635ca6ffd9978bf658c5f4e89a5f> (0)
  *   A teste/Spotify/src/app/faq/faq.component.html<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-f4c1c52fcb9d94b3d0a56f0946d43f3d> (1)
  *   A teste/Spotify/src/app/faq/faq.component.spec.ts<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-62eda705dc978a2257ee2b0c6ea7dc95> (25)
  *   A teste/Spotify/src/app/faq/faq.component.ts<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-ab2c8ccd6986fa2be9d6c0ff3c08ba33> (15)
  *   A teste/Spotify/src/app/footbar/footbar.component.css<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-f9e647100ed6001c69cdd0a240b7266e> (4)
  *   A teste/Spotify/src/app/footbar/footbar.component.html<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-76f7c3ed9f5b6029f104e3c2ed1516b8> (5)
  *   A teste/Spotify/src/app/footbar/footbar.component.spec.ts<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-37a96ef1f934b9f0f092fa2579a577be> (25)
  *   A teste/Spotify/src/app/footbar/footbar.component.ts<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-2d713f402517ee7c39f7750e0b469b4f> (21)
  *   M teste/Spotify/src/app/nav-bar/nav-bar.component.html<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-003cb0bbe337061bd5fef43986626006> (29)
  *   M teste/Spotify/src/app/show-case/show-case.component.css<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-9016b68a8a9fef749def10df71e10e4a> (75)
  *   M teste/Spotify/src/app/show-case/show-case.component.html<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-e8fdd4c6230335d7189a62f0c6ee1a2c> (41)
  *   M teste/Spotify/src/styles.css<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-a1653514497c3dce1488e473c97ab7fb> (16)

Patch Links:

  *   https://github.com/lucwars/Desenvolvimento-web2/pull/1.patch
  *   https://github.com/lucwars/Desenvolvimento-web2/pull/1.diff

—
You are receiving this because your review was requested.
Reply to this email directly, view it on GitHub<https://github.com/lucwars/Desenvolvimento-web2/pull/1>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AGLHUTGR5D5J72B7KB3NSV3RHOGIXANCNFSM4LJESHEA>.

lucwars commented:
Ei lucas me passa seu whats

Enviado do Email<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows 10

________________________________
De: Lucas Magalhães do Nascimento <notifications@github.com>
Enviado: Saturday, March 14, 2020 11:13:31 AM
Para: lucwars/Desenvolvimento-web2 <Desenvolvimento-web2@noreply.github.com>
Cc: LUCAS ALVES <lucwars@hotmail.com>; Review requested <review_requested@noreply.github.com>
Assunto: [lucwars/Desenvolvimento-web2] footbar e faq (#1)


Gente, fiz o footbar com a data atual e criei uma rota para o faq ao clicar em Ajuda na navbar.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/lucwars/Desenvolvimento-web2/pull/1

Commit Summary

  *   footbar e faq

File Changes

  *   M teste/Spotify/package-lock.json<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-d6cd016436097c4e49a4f2abf430edbb> (58)
  *   M teste/Spotify/package.json<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-433a21331cf187b2b77ede3cd82d2d04> (4)
  *   M teste/Spotify/src/app/app-routing.module.ts<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-5a38fb200a90a6ca8e82fff2b3597d89> (15)
  *   M teste/Spotify/src/app/app.component.html<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-cf07c06a346366939b7576a913d6eb19> (3)
  *   M teste/Spotify/src/app/app.module.ts<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-c598003e335a06d9fd56a4a3f1f19189> (6)
  *   A teste/Spotify/src/app/faq/faq.component.css<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-e4be635ca6ffd9978bf658c5f4e89a5f> (0)
  *   A teste/Spotify/src/app/faq/faq.component.html<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-f4c1c52fcb9d94b3d0a56f0946d43f3d> (1)
  *   A teste/Spotify/src/app/faq/faq.component.spec.ts<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-62eda705dc978a2257ee2b0c6ea7dc95> (25)
  *   A teste/Spotify/src/app/faq/faq.component.ts<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-ab2c8ccd6986fa2be9d6c0ff3c08ba33> (15)
  *   A teste/Spotify/src/app/footbar/footbar.component.css<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-f9e647100ed6001c69cdd0a240b7266e> (4)
  *   A teste/Spotify/src/app/footbar/footbar.component.html<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-76f7c3ed9f5b6029f104e3c2ed1516b8> (5)
  *   A teste/Spotify/src/app/footbar/footbar.component.spec.ts<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-37a96ef1f934b9f0f092fa2579a577be> (25)
  *   A teste/Spotify/src/app/footbar/footbar.component.ts<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-2d713f402517ee7c39f7750e0b469b4f> (21)
  *   M teste/Spotify/src/app/nav-bar/nav-bar.component.html<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-003cb0bbe337061bd5fef43986626006> (29)
  *   M teste/Spotify/src/app/show-case/show-case.component.css<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-9016b68a8a9fef749def10df71e10e4a> (75)
  *   M teste/Spotify/src/app/show-case/show-case.component.html<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-e8fdd4c6230335d7189a62f0c6ee1a2c> (41)
  *   M teste/Spotify/src/styles.css<https://github.com/lucwars/Desenvolvimento-web2/pull/1/files#diff-a1653514497c3dce1488e473c97ab7fb> (16)

Patch Links:

  *   https://github.com/lucwars/Desenvolvimento-web2/pull/1.patch
  *   https://github.com/lucwars/Desenvolvimento-web2/pull/1.diff

—
You are receiving this because your review was requested.
Reply to this email directly, view it on GitHub<https://github.com/lucwars/Desenvolvimento-web2/pull/1>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AGLHUTGR5D5J72B7KB3NSV3RHOGIXANCNFSM4LJESHEA>.

Commit:
faq

lucsblume committed:
Footer Completo.

falta adicionar FAQ e data atual!

Commit:
routes

lucsblume committed:
FAQ

adicionando transições, hover verde, alterando fonte.

lucsblume committed:
Adicionando features

alterando Icones de playsLists

lucsblume committed:
Adicionando features 2

adicionado footer ao component do app.

lucsblume committed:
Formatação

lucsblume committed:
Adicionando features 3

adicionado logo do spotify no topo da nav-bar

Commit:
updated code

lucsblume committed:
Criando Cards de PlayLists

criação dos cards das bandas

Commit:
form css & html

lucsblume committed:
Updates nos cards

atualizando interface dos cards, inserindo animações e links. e margens no footer e no wrapper dos cards de musicas

lucsblume committed:
Updates nos cards

atualizando interface dos cards, inserindo animações e links. e margens no footer e no wrapper dos cards de musicas, 90% finalizado

lucsblume committed:
Merge remote-tracking branch 'origin/lucas' into lucas

lucsblume committed:
Aumentanto o numero de playlists

lucsblume committed:
Adicionando imagens nos cards

adicionando imagens dos albuns e botao para linkar com a play list de musicas

lucsblume committed:
Adicionando imagens 

adicionando imagens dos albuns e botao para linkar com a play list de musicas

lucsblume committed:
Merge remote-tracking branch 'origin/lucas' into lucas

Commit:
email validation

Commit:
form validation

Commit:
form validation

Commit:
form validation

Commit:
Serviço criado.

Commit:
Serviço criado.

Commit:
onSubmmit apenas com campos válidos

lucsblume committed:
Adicionando header a playlist.

Commit:
serviço 2.0

lucsblume committed:
Iniciando Component de audio

inserindo musicas e player de audio

Commit:
...

Commit:
Merge branch 'lucas' of https://github.com/lucwars/Desenvolvimento-web2 into lucas

lucsblume committed:
AudioPlayer semi completo

incluindo imagem de artista e algumas musicas testes

lucsblume committed:
Adicionando musicas e audios da playlist ALOK

ajustando os nomes das musicas, e margens

lucsblume committed:
Adicionanto icone de notas musicais

lucsblume committed:
Adicionado todas as musicas, adicionado nav bar de musicas.

adicionado window.scrollto para coordenar a posição de cada playlist de audio, adicionado instruções de voltar ao topo, adicionado botões neon

lucsblume committed:
Adicionando MOCK e modificações

adicionando diretorios mock e classes typescript do mock, fazendo modificações no botão neon (aumentando o margin-bottom).

lucsblume committed:
Adicionando diretorios de imagens e musicas

Commit:
Mocks

Commit:
Login

Commit:
user update

Commit:
in app memory api

Commit:
In Memmory Web Api funcionando

Commit:
começando o perfil do usuário

Commit:
Pagina de login e delete user funcionando

Commit:
usando localstorage

Commit:
delete user funcionando

Commit:
update de usuário funcionando

Commit:
user

Commit:
playlists usando DB

Commit:
pesquisa de musicas

Commit:
busca de musicas

Commit:
criação de playlists do usuario - BUGADA

Commit:
remoção de musica da playlist

lucsblume committed:
Implementando CSS

aplicando css na playlist, nos audios player, na playlist do usuario e no banco de busca, ajustando as imagens dos albuns e o tempo de cada musica

lucsblume committed:
Implementando CSS

aplicando css na playlist, nos audios player, na playlist do usuario e no banco de busca, ajustando as imagens dos albuns e o tempo de cada musica

*OBS: bug ainda não resolvido.

lucsblume committed:
Merge remote-tracking branch 'origin/lucas' into lucas

lucsblume committed:
Css border-radius

lucsblume committed:
Ajustando css da playlist de usuario

Commit:
update user

Commit:
adicionando numa playlist especifica

lucsblume committed:
Ajustando edit playlist do usuario

adicionado css moderado

Commit:
teste

Commit:
usando backend

Commit:
ajeitando integração com o backend

Commit:
terminando integração com o backend

lucsblume committed:
Ajustando Album BTS
